### PR TITLE
Updates to bayesian estimation fit docs

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**
Updates and expands the Bayesian estimation documentation and minor edits to formatting otherwise.

### CODE REVIEW

**Comment by pkofod** [comment]
I'll review it, thanks.

File: docs/src/basics/estimation.md [hunk 1]
```diff
@@ -24,9 +27,9 @@ then the value has an implicit standard normal interpretation, which amounts
 to having the L2 Euclidian distance taken as the likelihood during fitting
 procedures.
 
-**Note: Currently only `BayesMCMC` and `LaplaceI` support multiple output series.
+<!-- **Note: Currently only `BayesMCMC` and `LaplaceI` support multiple output series.
 all other likelihood approximations must have the derived and observation data
-under the name `dv`**
+under the name `dv`** Is this still true? -->
```
Comment (vjd) [inline docs/src/basics/estimation.md:32]: no, that is not true anymore. We support multiple `dv` in all algorithms

File: docs/src/basics/estimation.md [hunk 2]
```diff
@@ -24,9 +27,9 @@ then the value has an implicit standard normal interpretation, which amounts
 to having the L2 Euclidian distance taken as the likelihood during fitting
 procedures.
 
-**Note: Currently only `BayesMCMC` and `LaplaceI` support multiple output series.
+<!-- **Note: Currently only `BayesMCMC` and `LaplaceI` support multiple output series.
 all other likelihood approximations must have the derived and observation data
-under the name `dv`**
+under the name `dv`** Is this still true? --> 
 
 **Note: Currently the estimation procedures require that there only exists a
```
Comment (vjd) [inline docs/src/basics/estimation.md:34]: This is not true, right @pkofod

File: docs/src/basics/estimation.md [hunk 3]
```diff
@@ -121,26 +124,46 @@ field `individual`. For more information on the diagnostics, please see the
...
 
 ### BayesMCMCResults
 
 The MCMC chain is stored in the `chain` field of the returned `BayesMCMCResults`.
+Additionally the result can be converted into a `Chains` object from MCMCChains.jl,
+allowing utlilization of diagnostics and visualization tooling. This is discussed further in
+the Bayesian Estimation tutorial. 
+
 The following functions help with querying common results on the Bayesian
 posterior:
```
Comment (vjd) [inline docs/src/basics/estimation.md:169]: @Vaibhavdixit02 can we also add simple ways of extracting the chains, what can we do with the chain etc, or do you want to re-direct the user to the tutorials?

File: docs/src/basics/estimation.md [hunk 4]
```diff
@@ -121,26 +124,46 @@ field `individual`. For more information on the diagnostics, please see the
...
+    progress = Base.is_interactive,
+    kwargs...
+ )
 ```
 
+!!! info
+    
+    We use a `NUTS` with the generalised no-U-turn termination criterion and multinomial sampling on Hamiltonian system 
+    whose kinetic energy is specified with a diagonal metric (diagonal matrix with positive diagonal entries).
+    For numerical intergation of the Hamitlonian system, we use the ordinary leapfrog integrator. The adaptation is done
```
Comment (Vaibhavdixit02) [inline docs/src/basics/estimation.md:143]:
```suggestion
    For numerical intergation of the Hamiltonian system, we use the ordinary leapfrog integrator. The adaptation is done 

```

File: docs/src/basics/estimation.md (hunk 1 above)
Comment (Vaibhavdixit02) [reply docs/src/basics/estimation.md:32]: Yeah that's what I thought.

File: docs/src/basics/estimation.md (hunk 3 above)
Comment (Vaibhavdixit02) [reply docs/src/basics/estimation.md:170]: I feel that is better suited for the tutorial and we already have it written there so better to not populate the docs too much

Comment (Vaibhavdixit02) [reply docs/src/basics/estimation.md:170]: >the Bayesian Estimation tutorial. 
We should have a link here

Comment (pkofod) [reply docs/src/basics/estimation.md:170]: Yes, if you could place a link here that would be great.

Comment (Vaibhavdixit02) [reply docs/src/basics/estimation.md:174]: We should probably merge https://github.com/PumasAI/PumasTutorials.jl/pull/48 to get a stable link

**Comment by andreasnoack** [comment]
Could you please rebase this?

**Comment by Vaibhavdixit02** [comment]
LGTM?

**Pull request merged**